src/ucp/api/ucp.h

```diff
@@ -409,6 +425,33 @@ typedef enum {
 } ucp_stream_recv_flags_t;
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief flags to indicate types of events on a stream associated endpoint.
+ *
+ * This enumeration defines types of events on a stream associated endpoint.
+ * These flags will be set in the field @ref ucp_stream_poll_ep_t::flags of out
+ * parameter @a poll_eps returned by @ref ucp_stream_worker_poll routine.
+ */
```

Comment:
suggested re-wording:This enumeration defines the possible events that can be raised on a stream associated endpoint. These flags will be set in the `@ref` ucp_stream_poll_ep_t::flags field of the `@a` poll_eps array returned by the `@ref` ucp_stream_worker_poll routine.